### ISSUE TITLE
pd canvas size bug

### ISSUE THREAD

**Issue by 60-hz** [sebshader/pdnext#2]
Step to reproduce:

1 - create an object [pd sub]
2 - close it
3 - click in action mode to open it
4 - The patch size is very small
Same happens with abstractions

Also There is many other little ui bug with ceammc, It might be related to tcl/tk 8.6.11 which is too new for for pd 0.51-4 

OSX 10.15.4

![Capture d’écran 2021-03-14 à 23 36 00](https://user-images.githubusercontent.com/5449719/111086762-59f9fa80-851e-11eb-8bac-1f2810755e17.png)

**Comment by sebshader** [comment]
@60-hz odd, I found this bug awhile ago but thought I fixed it, and can't reproduce it on osx 10.14.6.. are you sure you're running the right pd-next version?
here is the pr it was supposed to be fixed in: https://github.com/pure-data/pure-data/pull/1236
it was merged into pd in 6fbe93

I'm not sure how to go about figuring this out since my osx version is very similar.. hmm..
I suppose the most I can do is to suggest looking at (right click on app) show package contents > Contents > Resources > tcl > pd_bindings.tcl and make sure that the lines 350 and 351 are as in the "files changed" section of the commit:

```
set width [winfo width $mytoplevel]
set height [winfo height $mytoplevel]
```
also if you don't mind confirming Pd>about pd says Pd 0.51.4 at the top of the app with this problem, just to make sure..

**Comment by 60-hz** [comment]
Oh, I checked and you are right, I think that I copied some old .tcl files from an old pd and forgot about it.
Sorry for the noise!
